ansible-fmw/templates/WDT_SSO_template-baseline.yaml.j2

```diff
@@ -0,0 +1,271 @@
+domainInfo:
+    AdminUserName: '{{ WEBLOGIC_USERNAME }}'
+    AdminPassword: '{{ WEBLOGIC_PASSWORD }}'
+topology:
+    Name: '{{ DOMAIN_NAME }}'
+    DomainVersion: 12.2.1.3.0
+    ProductionModeEnabled: true
+    NMProperties:
+        JavaHome: '{{ JAVA_HOME }}'
+        PropertiesVersion: 12.2.1.3.0
+        weblogic.StartScriptName: startWebLogic.sh
+    Server:
+        AdminServer:
+            Machine: oam_Machine_1
+            WebServer:
+                WebServerLog:
+                    LogFileFormat: extended
+                    ELFFields: date time cs-method ctx-ecid ctx-rid cs-uri sc-status
+                        bytes
+        oam_server1:
```

Comment:
those playbooks are meant _only_  for `WLS`   
why replace it with `OAM`?  this will lead to unwanted complexity.